### PR TITLE
Pass on tutorial for code example include-ability and reduced verbosity

### PR DESCRIPTION
**Pull request by schrockn**
## Summary & Motivation

Making code examples include all imports and eliminating direct usage of AssetKey for verbosity reduction.

## How I Tested These Changes

Read

## Changelog [New | Bug | Docs]

NOCHANGELOG

### CODE REVIEW

**Comment by schrockn** [comment]
* **#23923** <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/23923?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `master`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @schrockn and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: examples/experimental/dagster-airlift/README.md [hunk 1]
```diff
@@ -265,14 +272,15 @@ defs = build_defs_from_airflow_instance(
 
 For all other operator types, we recommend creating a new factory class whose arguments match the inputs to your Airflow operator. Then, you can use this factory to build definitions for each Airflow task.
 
-For example, our `load_raw_customers` task uses a custom `LoadCSVToDuckDB` operator. We'll define a `CSVToDuckDBDefs` factory to build corresponding software-defined assets:
+For example, our `load_raw_customers` task uses a custom `LoadCSVToDuckDB` operator. We'll define a function  `load_csv_to_duckdb_defs` factory to build corresponding software-defined assets:
```
Comment (dpeng817) [inline examples/experimental/dagster-airlift/README.md:275]: Another pass on this sentence would be nice. Feels like it throws forward a lot of verbiage in a slightly confusing way

File: examples/experimental/dagster-airlift/README.md [hunk 2]
```diff
@@ -130,20 +130,22 @@ The first and third tasks involve a single table each. We can manually construct
...
+
+from dagster_airlift.core import build_defs_from_airflow_instance, dag_defs, task_defs
+
 defs = build_defs_from_airflow_instance(
     airflow_instance=airflow_instance,
     defs=dag_defs(
         "rebuild_customers_list",
         task_defs(
             "load_raw_customers",
-            AssetSpec(key=AssetKey(["raw_data", "raw_customers"]))
```
Comment (dpeng817) [inline examples/experimental/dagster-airlift/README.md:139]: Why is the "raw_data" piece even necessary here? Is this an artifact of dbt?

**Review by dpeng817 — CHANGES_REQUESTED** [review]
Some small comments and a question

File: examples/experimental/dagster-airlift/README.md (hunk 2 above)
Comment (schrockn) [reply examples/experimental/dagster-airlift/README.md:139]: Not sure. I wasn't really examining the core content here, just cleaning up what is there already.

**Review by dpeng817 — APPROVED** [review]

File: examples/experimental/dagster-airlift/README.md (hunk 1 above)
Comment (schrockn) [reply examples/experimental/dagster-airlift/README.md]: per offline discussion I was just doing a light pass here, rather than working on core content.

**Pull request merged**